docs/middleware.rst

```diff
@@ -264,22 +263,27 @@ To add or remove items in different layers, use the following API:
         >>> assert w3_2.middleware_onion.get('test_middleware')
 
 
-Optional Middleware
--------------------
-
-Web3 ships with non-default middleware, for your custom use. In addition to the other ways of
-:ref:`Modifying_Middleware`, you can specify a list of middleware when initializing Web3, with:
+Instantiate with Custom Middleware
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        
+Instead of working from the default list, you can also specify a custom list of
```

Comment:
nit. Sort of implies you can specify middleware in addition to the default, rather than replacing - which is actually a separate feature via `external_middleware`.

```suggestion
Instead of working from the default list, you can specify a custom list of
```